postgresql-native/build.gradle

```diff
@@ -30,9 +30,15 @@ configurations {
 
 dependencies {
     jacocoRuntime "org.jacoco:org.jacoco.agent:${jacoco.toolVersion}:runtime"
-    compile "org.postgresql:postgresql:${postgreSQLDriverVersion}"
+
     checkstyle project(":build-config:checkstyle")
     checkstyle "com.puppycrawl.tools:checkstyle:${puppycrawlCheckstyleVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-runtime', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'sql-native', version: "${stdlibSqlVersion}"
+    compile group: 'org.ballerinalang', name: 'time-native', version: "${stdlibTimeVersion}"
+    implementation group: 'org.postgresql', name: 'postgresql', version: "${postgreSQLDriverVersion}"
```

Comment:
is this Postgres driver dependency?

Reply:
Yes, It is dependency.
https://github.com/ballerina-platform/module-ballerinax-postgresql/blob/main/postgresql-native/src/main/java/module-info.java#L24